drivers/src/hal_sfc.c

```diff
@@ -531,12 +519,15 @@ void spiflash_cmd_write(struct spi_flash *flash, const u8_t * cmd,
 	spiflash_read_write(cmd_desc, cmd_length, NULL);
 }
 
-int spiflash_cmd_poll_bit(struct spi_flash *flash, unsigned long timeout,
-		u8_t cmd, u32_t poll_bit, u32_t bit_value)
+__ramfunc int spiflash_cmd_poll_bit(struct spi_flash *flash,
+	unsigned long timeout, u8_t cmd, u32_t poll_bit, u32_t bit_value)
 {
 	u32_t status = 0;
+	u32_t count	 = 300000;
```

Comment:
Can you add some comments to describe what does this number mean?

Reply:
fixed